### PR TITLE
golang 1.12, increment module versions

### PR DESCRIPTION
**Pull request by tchughesiv**
Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,38 +9,44 @@ else
        Q = @
 endif
 
-#VERSION = $(shell git describe --dirty --tags --always)
-#REPO = github.com/RHsyseng/console-cr-form
+# VERSION = $(shell git describe --dirty --tags --always)
+# REPO = github.com/RHsyseng/console-cr-form
 
-#export CGO_ENABLED:=0
+export GO111MODULE = on
```
Comment (bmozaffa) [inline Makefile:15]: I don't think you need this with 12+ versions of Go. Want to double check?

File: frontend/dist/index.html [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-  <div id="mount"></div>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8" />
+  <script type="text/javascript" src="/bundle.js"></script></head>
+  <body>
+    <div id="mount"></div>
+  </body>
+</html>
```
Comment (bmozaffa) [inline frontend/dist/index.html:9]: As you're cleaning up, the missing line breaks at the end of some files is causing a lot of unnecessary diff markers.

Comment (tchughesiv) [reply frontend/dist/index.html:9]: i didn't make this change... trying to figure out why it's happening

File: Makefile (hunk 1 above)
Comment (tchughesiv) [reply Makefile:15]: 1.13+ is when it's no longer needed - https://golang.org/doc/go1.13#modules

File: frontend/dist/index.html (hunk 2 above)
Comment (bmozaffa) [reply frontend/dist/index.html]: Right, I know. When there is no longer break at the end of the file, that line shows up when there is a change anywhere else in the file.

**Comment by tchughesiv** [comment]
closing in favor of go 1.13 @ https://github.com/RHsyseng/console-cr-form/pull/121
